randovania/games/samus_returns/logic_database/header.txt

```diff
@@ -104,10 +92,8 @@ Templates
                           # Beam Burst
                           Beam Burst
                           Any of the following:
-                              Aeion ≥ 1600 and Shoot Wave Beam
-                              All of the following:
-                                  Aeion ≥ 1100
-                                  Plasma Beam or Shoot Spazer Beam
+                              Aeion ≥ 1600 and Wave Beam
+                              Aeion ≥ 1100 and All Beams
```

Comment:
this behaviour is different now.
Previous plasma is all beams. But previous shoot spazer is now missing.

Reply:
For now I'll just leave this since the difference between the two is having Wave. I'll add a TODO.

Reply:
Actually it looks like I changed the Gamma+ so ignore lol